PseudoLocalize.Tests/EndToEndTest.cs

```diff
@@ -20,5 +20,22 @@ public async Task ShouldLeaveTheXmlUntouchedWhenAlreadyTransformed()
             // Assert
             Assert.That(File.ReadAllBytes(fileName), Is.EqualTo(original), "The input file has changed.");
         }
+
+        [Test]
+        public async Task ShouldChangeTheLengthenCharacter()
+        {
+            // Arrange
+            string inputFileName = Path.GetFullPath("not-localized.xlf");
+            string outputFileName = Path.GetFullPath("not-localized.qps-Ploc.xlf");
+            byte[] original = await File.ReadAllBytesAsync(inputFileName);
+
+            // Act
+            Program.Main(new[] { inputFileName, "--lengthen-char", "." });
+
+            // Assert
+            Assert.That(File.ReadAllBytes(inputFileName), Is.EqualTo(original), "The input file has changed.");
+            Assert.That(File.ReadAllBytes(outputFileName), Is.Not.EqualTo(outputFileName), "The output file has not changed.");
+            Assert.That(File.ReadAllText(outputFileName), Contains.Substring(">[Åñýţĥîñĝ···]<"), "The specified lengthen character was not used.");
```

Comment:
It's strange that you specify "." in the command line but get "·" in the results.

Reply:
That's because the `'.'` goes through the [Accents](https://github.com/martincostello/Pseudolocalizer/blob/fd64a298e86329c5f6f6dbbb65a7583900fe4e14/PseudoLocalizer.Core/Accents.cs#L27) transformer after it's lengthened by default, so becomes `'·'`. You'd get `'.'` if that wasn't used.

Reply:
I see. That may be unclear to the users, as the argument's description doesn't point on that. But I suppose it would be difficult to use the specified symbol as is...

Reply:
I've updated the description for the parameter in 90bcb93f7e360fc959269dd415672fb7453b4bb6, do you think that's sufficient?